package.json

```diff
@@ -113,9 +116,11 @@
     "truffle-plugin-verify": "^0.3.10",
     "ts-node": "^8.8.1",
     "typechain": "^1.0.5",
+    "typechain-target-trufflehotfix": "^0.0.4-alpha",
```

Comment:
I'm assuming that `@openeth/truffle-typings` is related to this typechain target?